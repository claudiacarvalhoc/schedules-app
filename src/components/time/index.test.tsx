import React from 'react';
import Time from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { TimeProps } from './index';

const getShallowWrapper = (props: TimeProps) => {
    return shallow(
        <Time {...props} />
    );
};

describe('<Time />', () => {
    it('render correctly', () => {
        const props = {
            label: 'Start',
            time: 'April 28 2021',
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
});
