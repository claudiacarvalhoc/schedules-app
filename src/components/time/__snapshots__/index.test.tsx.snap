// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Time /> render correctly 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    className="label"
    gutterBottom={true}
    variant="overline"
  >
    Start
  </WithStyles(ForwardRef(Typography))>
  <div>
    <WithStyles(ForwardRef(Chip))
      label="April 28 2021"
      size="small"
    />
  </div>
</div>
`;
